docs/DeveloperGuide.md

```diff
@@ -952,7 +955,33 @@ testers are expected to do more *exploratory* testing.
 
     1. Other incorrect add commands to try: `add n/James! <other valid params>`, `add <one invalid param amongst other valid params>`, `...`.<br>
        Expected: Similar to previous.
-       
+
+### Find a contact
+1. Find a contact by name
+
+    1. Prerequisites: Multiple contacts in the list with at least one contact named Roy.
+
+    1. Test case: `find n/Roy`<br>
+       Expected: Display contacts with name Roy. Number of contacts found shown in the status message.
+    1. Test case: `find n/roy!`<br>
+       Expected: No changes made. Error details shown in the status message.
+
+    1. Other incorrect untag commands to try: `find`, `find n/<non existent name>`, `...`.<br>
```

Comment:
I dont think find n/non existent name will be same as previous because it shouldn;t show an error right and should show 0 person listed

Reply:
You are right here. Thanks!